pom.xml

```diff
@@ -258,12 +258,12 @@
 			<dependency>
 				<groupId>org.hibernate</groupId>
 				<artifactId>hibernate-validator</artifactId>
-				<version>4.2.0.Final</version>
+				<version>6.1.0.Alpha4</version>
```

Comment:
can you look for the latest full release , we dont depend onalpha or beta releases

Reply:
This also I have done it, however I must say the post on talk was misleading, because it was pointing to the alpha releases. But I got it and I have worked on it already though the build is still failing locally.